bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/SecurityApi.java

```diff
@@ -78,16 +81,26 @@ private Collection<HomeEvent> getEvents(@Nullable Object... params) throws Netat
         throw new NetatmoException("home should not be null");
     }
 
-    public Collection<HomeEvent> getHomeEvents(String homeId) throws NetatmoException {
-        return getEvents(PARAM_HOME_ID, homeId);
+    public List<HomeEvent> getHomeEvents(String homeId, @Nullable ZonedDateTime oldestKnown) throws NetatmoException {
+        List<HomeEvent> events = getEvents(PARAM_HOME_ID, homeId);
+
+        // we have to rewind to the latest event just after oldestKnown
+        HomeEvent oldestRetrieved = events.get(events.size() - 1);
```

Comment:
Is the API documented that returned events will be sorted form the most recent to the last recent ?

Reply:
Yes, events are returned from the most recent to the oldest.